file_paths.md

```diff
@@ -292,6 +294,7 @@ create a `puppet` user or group.
             puppet *                          # ruby bin wrapper
         lib *
             puppet.rb *
+        locale *
```

Comment:
I also see that if we call `msgfmt` - I see a reference in the i18n build workflow doc at https://davesteele.github.io/debian/development/2015/12/10/i18n-build-flow/ to put them in `/usr/share/locale/<language>[-<COUNTRY>]/LC_MESSAGES/<package name>.mo`

But I don't know if there's any reason for that, etc.